sakai-root/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.sakaiproject</groupId>
+    <artifactId>master</artifactId>
+    <version>24-SNAPSHOT</version>
+    <relativePath>../master/pom.xml</relativePath>
+  </parent>
+
+  <name>Sakai Root</name>
+  <groupId>org.sakaiproject</groupId>
+  <artifactId>ROOT</artifactId>
```

Comment:
Adrian can you explain this artifact ROOT? do we have a better name?

Reply:
So the war file is called ROOT.war and is deployed to the tomcat root, that's all.

Reply:
Do you think I can change the artifact id while still creating ROOT.war to deploy to tomcat?

Reply:
the reason we have this sakai-root thing is that we need to deploy a single service worker to the top level of our hierarchy so it can control all of Sakai's paths. It's crucial.

Reply:
the artifact is now "serviceworker" and I've used deployId to deploy to tomcat as the ROOT war.